blog/integration-stack-4.4.0.Final.adoc

```diff
@@ -0,0 +1,94 @@
+= Production BRMS and Fuse Tooling for Neon
+:page-layout: blog
+:page-author: pleacu
+:page-date: 2016-11-17
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+
+*Try our complete Eclipse-Neon capable, Devstudio 10.1.0 compatible integration tooling.*
+
+.*JBoss Tools Integration Stack*
+image::/blog/images/jbosstools-jbdevstudio-blog-header.png[caption=""]
+
+*JBoss Tools Integration Stack 4.4.0.Final / JBoss Developer Studio Integration Stack 10.0.0.GA*
+
+NOTE: All of the Integratioon Stack components have been verified to work with the same dependencies as JBoss Tools 4.4 and Developer Studio 10.
+
+== What's new for this release?
+
+Fuse Tooling/SwitchYard and BRMS Tooling are released for the Eclipse/Neon platform (devstudio 10).  The latest link:http://access.redhat.com/documentation/en/red-hat-jboss-fuse/6.3/[Fuse runtime] and link:http://developers.redhat.com/products/brms/download/[BRMS runtime] are supported.  Also - we're happy to offer the latest Mars-based DataVirtualization bits as Early Access on our Neon platform.
```

Comment:
I replaced the `devstudio 10` text with a link to http://localhost:4242/downloads/devstudio/neon/10.1.0.GA.html#central instead.